vm/src/stdlib/itertools.rs

```diff
@@ -29,6 +29,15 @@ mod decl {
     use num_traits::{Signed, ToPrimitive};
     use std::fmt;
 
+    fn pickle_deprecation(vm: &VirtualMachine) -> PyResult<()> {
+        warnings::warn(
+            vm.ctx.exceptions.deprecation_warning,
+            "Itertool pickle/copy/deepcopy support will be removed in a Python 3.14.".to_owned(),
```

Comment:
```suggestion
            "Itertools pickle/copy/deepcopy support will be removed in a Python 3.14.".to_owned(),
```
Otherwise adding itertool to cspell dictionary will fix the CI failure